lib/skywalking/client.lua

```diff
@@ -21,12 +21,12 @@ local Client = {}
 
 -- Tracing timer reports instance properties report, keeps alive and sends traces
 -- After report instance properties successfully, it sends keep alive packages.
-function Client:startBackendTimer(backend_http_uri)
+function Client:startBackendTimer(backend_http_uri, in_current_process)
```

Comment:
How to set this? I remember Nginx shares the same configuration across different processes. How to make them using different?

Reply:
They are some ways in Nginx to communicate among processes, like using the shared memory, listening different port in different process, etc. And there is different ID to different worker process. So different Nginx process can be assigned different job.

Reply:
Yes, I know that question. My question is, this is a user API parameter, how a user is going to set this? Or what way do you expect to set?
There is no doc, use case, or example updated, I can't get the idea of adding this.